frontend/control-center/src/translations.ts

```diff
@@ -0,0 +1,887 @@
+      connectGoogle: 'Conectar los mensajes de Google Business',
+      googleConfigurationText:
+        'La fuente de mensajes de empresa de Google requiere la siguiente configuración para enviar mensajes a tu instancia de Airy Core:',
+      googleAccountKey: 'Una clave de cuenta de servicio de Google',
+      googleKey: 'Una clave de socio de Google',
+      googleConfigurationText2: `Compruebe la documentación de Airy`,
+      googleConfigurationText3: 'para más información.',
+
+      //Instagram
+
+      instagramAccount: 'ID de la página de Facebook conectada a la cuenta de Instagram',
+      instagramAccountPlaceholder: 'Añade el ID de la página de Facebook',
+      instagramAccountId: 'ID de la cuenta de Instagram',
+      instagramAccountIdPlaceholder: 'Añade el ID de la cuenta de Instagram',
+      connectInstagram: 'Conectar Instagram',
+      instagramConfigurationText: 'La fuente de Instagram requiere la siguiente configuración:',
+      instagramConfigurationText2:
+        'Un identificador y un secreto de la aplicación para que la plataforma pueda devolver los mensajes a través de tu aplicación de Instagram',
+      instagramConfigurationText3:
+        'Una integración de webhook para que la plataforma pueda ingerir mensajes de tus páginas de Instagram',
+      instagramConfigurationText4: 'Un token de página para cada página de Facebook que desee integrar',
+      instagramConfigurationText5: `Compruebe la documentación de Airy`,
+      instagramConfigurationText6: 'para más información.',
+
+      //Twilio
+
+      twilioPhoneNumber: 'Número de teléfono de Twilio',
+      twilioPhoneNumberPlaceholder: 'Número de compra +123456789',
+      connectSmsNumber: 'Conectar número Sms',
+      updateSmsNumber: 'Actualizar el número de Sms',
+
+      connectWhatsapp: 'Conectar con Whatsapp',
+      connectWithTwilio: 'Conéctese primero con Twilio',
+      twilioConfigurationText: 'Antes de conectar un número para SMS o Whatsapp, debes añadir un',
+      twilioConfigurationText2: 'Twilio Auth Token',
+      twilioConfigurationText3: 'a la',
+      twilioConfigurationText4: 'airy.yaml',
+      twilioConfigurationText5: 'archivo.',
+      twilioConfigurationText6: 'Después, tienes que comprar un número.',
+      twilioConfigurationText7: 'Consulte',
+      twilioConfigurationText8: `Compruebe la documentación de Airy`,
+      twilioConfigurationText9: 'para más información.',
+      connectWhatsappNumber: 'Conectar el número de Whatsapp',
+      updateWhatsappNumber: 'Actualizar el número de Whatsapp',
+
+      //Connectors
+
+      noResults: 'Resultado no encontrado.',
+      noResultsTerm: 'Intente buscar un término diferente.',
+      noConnectorsFound: 'No se han encontrado conectores',
+      noConnectorsFoundTerm: 'No tiene ningún conector instalado, por favor abra el',
+      noConnectorsFoundMore: 'y explorar más.',
+      installed: 'Instalado',
+      notInstalled: 'No Instalado',
```

Comment:
For instalado  / no instalado, I suggest we add 'Instalado(s)' and 'No Instalado(s)' to account for singular and plural cases.